sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -132,14 +133,15 @@ def finish(self):
         """Set the end time for a span."""
         self.span_instance.end()
 
-    def to_header(self):
+    @staticmethod
+    def to_header():
```

Comment:
You can't change this signature, since it's coming from your inheritance. Add a pylint ignore if necessary.
I'm surprised you don't get a pylint warning saying you don't inherit your base class

Reply:
Well, you *can* change this method to static. I'm not sure if it is a good idea or not. 

@lmazuel, what are the corner cases you are concerned with here? You can still call a `@staticmethod` through an instance. Can you give an example of where this would cause actual problems (and it is not just being weird)?

Reply:
No, all examples are weird, like:
```python
foo = Foo()
Foo.to_header(foo)
```

I'm just usually picky on subclassing :p